test/controllers/DashboardControllerSpec.scala

```diff
@@ -39,23 +67,76 @@ class DashboardControllerSpec extends FrontEndTestHelper {
     "render the judgment dashboard page with an authenticated judgment user" in {
       val controller = new DashboardController(
         getAuthorisedSecurityComponents,
-        getValidJudgmentUserKeycloakConfiguration)
-      val dashboardPage = controller.dashboard().apply(FakeRequest(GET, "/dashboard"))
+        getValidJudgmentUserKeycloakConfiguration,
+        consignmentService)
+      val dashboardPage = controller.dashboard().apply(FakeRequest(GET, "/dashboard").withCSRFToken)
       status(dashboardPage) mustBe OK
       contentType(dashboardPage) mustBe Some("text/html")
-      contentAsString(dashboardPage) must include ("Judgment Transfer Welcome")
+
+      contentAsString(dashboardPage) must include ("Welcome")
       contentAsString(dashboardPage) must include ("Welcome to the Transfer Digital Records service")
-      contentAsString(dashboardPage) must include ("Upload your judgment to start a new transfer")
-      contentAsString(dashboardPage) must include ("Placeholder page")
+      contentAsString(dashboardPage) must include ("Upload your records to start a new transfer")
+      //Judgment and standard dashboard pages are visually the same so check for the create consignment form.
+      contentAsString(dashboardPage) must include("""<form action="/dashboard"""")
     }
 
     "return a redirect to the auth server with an unauthenticated user" in {
       val controller = new DashboardController(
         getUnauthorisedSecurityComponents,
-        getValidKeycloakConfiguration)
+        getValidKeycloakConfiguration,
+        consignmentService)
       val dashboardPage = controller.dashboard().apply(FakeRequest(GET, "/dashboard"))
       redirectLocation(dashboardPage).get must startWith("/auth/realms/tdr/protocol/openid-connect/auth")
       status(dashboardPage) mustBe FOUND
     }
   }
+
+  "DashboardController POST" should {
+    "create a new consignment" in {
```

Comment:
Maybe reword the text of the test to make it more specific: _create a new consignment for a judgment user_